src/schemas/json/dependabot-2.0.json

```diff
@@ -451,6 +451,7 @@
         "Europe/Kaliningrad",
         "Europe/Kiev",
         "Europe/Kirov",
+        "Europe/Kyiv",
```

Comment:
Isn't this captured above in `Europe/Kiev`?

Reply:
Both are valid. `"Europe/Kiev"` is now an accepted value, as found in the following links provided in #3508:

* dependabot/dependabot-core#6132
* [List of tz database timezones, Europe/Kyiv](https://en.wikipedia.org/w/index.php?title=List_of_tz_database_time_zones&oldid=1195226764#KYIV)

`"Europe/Kiev"` is a link. `"Europe/Kyiv"` is canonical in the database. [Dependabot's documentation for timezones](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#scheduletimezone) states:

> The time zone identifier must be from the Time Zone database maintained by IANA. See <link to Wikipedia "List of tz database timezones" page, same as above>